public/app/features/search/page/SearchPage.tsx

```diff
@@ -177,6 +198,14 @@ export default function SearchPage() {
           <InlineField label="Show manage options">
             <InlineSwitch value={showManage} onChange={() => setShowManage(!showManage)} />
           </InlineField>
+          <InlineField label="Folder">
+            <Select
```

Comment:
yes, this is uncontrolled.  For this usage, it is OK because we do not set the folder value from anywhere else.